functions/v2/pubsub/subscribe-to-topic/src/main/java/functions/eventpojos/PubSubBody.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.cloud.functions.v2.eventpojos;
+
+import java.util.Map;
+
+public class PubSubBody {
+  private PubSubMessage message;
```

Comment:
Can we import `PubSubMessage` from the CloudEvents SDK by chance? (Import [here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/functions/pubsub/subscribe-to-topic/src/main/java/functions/SubscribeToTopic.java#L23), dependency add [here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/functions/pubsub/subscribe-to-topic/pom.xml#L48-L53).)

Reply:
Adding it in!

Reply:
These Libraries aren't GA and we shouldn't use them yet.

Reply:
Un-adding them in!